moonshot/src/redteaming/attack/attack_module.py

```diff
@@ -76,6 +79,11 @@ def check_stop_condition(self):
         """
         pass
 
+    def check_stop_condition_iteration(
+        self,
+    ) -> bool:
```

Comment:
can have the doc strings here and whether is this an abstract class or something?

Reply:
forgot to remove. will just use the check_stop_condition function